documentation/modules/metrics/ref_metrics-config-files.adoc

```diff
@@ -5,50 +5,114 @@
 
 [id='ref-metrics-config-files-{context}']
 
-= Example Metrics files
+= Example metrics files
 
-You can find the example metrics configuration files in the `examples/metrics` directory.
+You can find example Grafana dashboards and other metrics configuration files in the `examples/metrics` directory.
 
+.Example metrics files provided with Strimzi
 [source]
 --
 metrics
-├── grafana-install
-│   ├── grafana.yaml <1>
-├── grafana-dashboards <2>
+├── grafana-dashboards <1>
+│   ├── strimzi-cruise-control.json
+│   ├── strimzi-kafka-bridge.json
 │   ├── strimzi-kafka-connect.json
-│   ├── strimzi-kafka.json
-│   ├── strimzi-zookeeper.json
+│   ├── strimzi-kafka-exporter.json <2>
```

Comment:
what's the rationale behind highlighting that it's a dashboard for Kafka Exporter but not saying the same for the bridge, cruise control, and so on? Maybe we should just remove this one?

Reply:
Exporter is in a separate section, so it was for the link really. But, you're right, I don't think we need it here after the restructure.
Removed.